activerecord/test/cases/integration_test.rb

```diff
@@ -1,3 +1,4 @@
+# coding: utf-8
```

Comment:
No, it no longer is w/ current rubies, but Emacs puts that in when it detects the need. I can remove it before merging.